src/tensorflow_backend_tf.h

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2019-2020, NVIDIA CORPORATION. All rights reserved.
+// Copyright 2020-2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
should this be 2019-2022?

Reply:
I think now this is tracking TF2 repo wrapper?

Reply:
yes, does that mean the tf2 repo copyright years are wrong?

Reply:
May be the wrapper for TF2 was written in 2020 and not in 2019. I would still prefer this to be an exact copy of the TF2 upstream.

Reply:
I checked and it has not been updated...I think we should at least fix the github one for this release and update the same on the tensorflow branch?